es/tutorials-and-examples/cms/installation.rst

```diff
@@ -0,0 +1,78 @@
+Tutorial de gestión de contenido
+###########
+
+Este tutorial lo guiará a través de la creación de una aplicacion simple: https://github.com/cakephp/docs/blob/3.0/en/tutorials-and-exampĺes/cms/installation.rst#id1. Para empezar, instalaremos CakePHP, crearemos nueestra propia base de datos y crearemos una administracion simple de artículos.
+
+Esto es lo que necesitarás:
+
+1) Un servidor de base de datos. En este tutorial vamos a utilizar MySQL. Necesitara saber lo suficiente sobre SQL para crear una base de datos y ejecutar fragmentos de SQL desde el tutorial. CakePHP se encargará de construir todas las consultas que su aplicación necesite. Ya que estamos usando MySQL, también asegúrese de tener pdo_mysql habilitado en PHP.
+
+2) Conocimientos básicos de PHP.
+
+Antes de comenzar, debe asegurarse de tener una vérsion actualizada de PHP:
+
+	php -v
```

Comment:
The various shell blocks you have are missing formatting. They should look like:

```rst
.. code-block:: bash

    php -v

Next line of text.
```